app/forms/PensionCustomerReferenceNumberForm.scala

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright 2022 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package forms
+
+import filters.InputFilters
+import forms.validation.StringConstraints.nonEmpty
+import forms.validation.mappings.MappingUtil.trimmedText
+import play.api.data.Form
+import play.api.data.validation.Constraint
+
+object PensionCustomerReferenceNumberForm extends InputFilters {
+
+  val pensionsCustomerReferenceNumberId: String = "pensionsCustomerReferenceNumberId"
```

Comment:
Given that it's used once, this could be inlined.

Reply:
The problem with inlining these variables is, the test cases use them.

Reply:
It's OK to use literals in the tests; in fact, it makes it easier to see the inputs.